map_model/src/edits/compat.rs

```diff
@@ -89,6 +90,13 @@ pub fn upgrade(mut value: Value, map: &Map) -> Result<PermanentMapEdits> {
             .unwrap()
             .insert("version".to_string(), Value::Number(8.into()));
     }
+    if value["version"] == Value::Number(8.into()) {
```

Comment:
oh nice! I haven't seen you change this migration script before and kind of forgot about it.

Reply:
These're slowly getting easier to write, as I figure out that modifying serde json values in-place is easier than consuming + transforming + appending.